packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/ResultTableView/index.js

```diff
@@ -1,11 +1,55 @@
 import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+
 import { createInjectSearch } from 'coreModules/search/higherOrderComponents'
+
+import { RowLayout } from 'coreModules/layout/components'
+import { injectWindowHeight } from 'coreModules/size/higherOrderComponents'
 import InfiniteTable from './InfiniteTable'
+import InfiniteTableHeader from './InfiniteTableHeader'
+
+const infiniteTable = {
+  key: 'infiniteTable',
+  renderRow: props => <InfiniteTable {...props} />,
+}
+
+const infiniteTableHeader = {
+  height: '50px',
+  key: 'infiniteTableHeader',
+  renderRow: props => <InfiniteTableHeader {...props} />,
+}
+
+const rows = [infiniteTableHeader, infiniteTable]
+
+const propTypes = {
+  availableHeight: PropTypes.string.isRequired,
+  mainColumnActiveTab: PropTypes.string.isRequired,
+  windowHeight: PropTypes.number.isRequired,
```

Comment:
`mainColumnActiveTab` and `windowHeight` are not used in this component, you can remove those props from here and in lines 35-36